src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -143,6 +183,78 @@ def post_process(self, outputs, target_sizes):
 
         return results
 
+    def post_process_one_shot_object_detection(self, outputs, threshold=0.6, nms_threshold=0.3, target_sizes=None):
+        """
+        Converts the output of [`OwlViTForObjectDetection.image_guided_detection`] into the format expected by the COCO
+        api.
+
+        Args:
+            outputs ([`OwlViTImageGuidedObjectDetectionOutput`]):
```

Comment:
The outputs here aren't an instance of `OwlViTImageGuidedObjectDetectionOutput`, I assume, since it's now implemented as a method.